packages/devtools_app/lib/src/screens/debugger/codeview_controller.dart

```diff
@@ -272,9 +274,9 @@ class CodeViewController extends DisposableController
           (last, e) => last..addAll(e.entries),
         ),
       );
-    } catch (e) {
+    } catch (e, st) {
       // Ignore - not supported for all vm service implementations.
-      log('$e');
+      _log.shout(e, e, st);
```

Comment:
should these be shouts or warnings if our intent is to ignore the exception?

Reply:
oops yes thanks for catching these.